java/ql/src/Security/CWE/CWE-023/PartialPathTraversalRemainder.inc.qhelp

```diff
@@ -26,10 +26,9 @@ and not just children of <code>parent</code>, which is a security issue.
 
 <p>
 
-In this example, the <code>if</code> statement checks if <code>parent.getCanonicalPath() + File.separator </code> 
-is a prefix of <code>dir.getCanonicalPath()</code>. Because <code>parent.getCanonicalPath() + File.separator</code> is
-indeed slash-terminated, the user supplying <code>dir</code> can only access children of 
-<code>parent</code>, as desired.
+In this example, the <code>if</code> statement checks if <code>parent.toPath()</code> 
+is a prefix of <code>dir.normalize()</code>. Because <code>Path#startsWith</code> does the correct check that 
+<code>dir</code> is a child of <code>parent</code>, users will not be able to access siblings of <code>parent</code>, as desired.
```

Comment:
```suggestion
In this example, the <code>if</code> statement checks if <code>parent.toPath()</code> 
is a prefix of <code>dir.normalize()</code>. Because <code>Path#startsWith</code> checks that 
<code>dir</code> is a child of <code>parent</code>, users will not be able to access siblings of <code>parent</code>, as desired.
```

Hi from Docs 👋 —Does this still make sense with this wording?